webpack/assets/javascripts/react_app/components/users/PersonalAccessTokens/PersonalAccessTokens.js

```diff
@@ -32,10 +33,18 @@ const PersonalAccessTokens = ({ url, canCreate }) => {
     dispatch(clearNewPersonalAccessToken());
 
   const boundRevokePersonalAccessToken = id => {
-    if (window.confirm(__('Do you really want to revoke Access Token?'))) {
-      dispatch(revokePersonalAccessTokenAction({ url, id }));
-    }
+    dispatch(
+      openConfirmModal({
+        label: __('Revoke'),
+        message: __('Do you really want to revoke Access Token?'),
+        onConfirm: closeCB => {
+          dispatch(revokePersonalAccessTokenAction({ url, id }));
```

Comment:
@sufanek1 hey, what do you think about this approach?

Reply:
why not :)